src/main/java/com/profittracker/ProfitTrackerOverlay.java

```diff
@@ -45,63 +46,61 @@ public Dimension render(Graphics2D graphics) {
         String titleText = "Profit Tracker:";
         long secondsElapsed;
         long profitRateValue;
+        if(ptConfig.ingameOverlay()) {
```

Comment:
Hey, I see this "IF" is the main change in the pull request. I think a nicer way to do this is
reversing the if:
`if (!ptConfig.ingameOverlay()) {
    return null;
}
`
Thus avoiding all the extra indentation.

Reply:
Oh you are totally right, I don't see why I didn't do that first, good idea. Well thanks for looking at my pull anyway (: it's my first time